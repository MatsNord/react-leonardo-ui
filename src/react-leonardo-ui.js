export { default as Button } from './button';
export { default as ButtonGroup } from './button-group';
export { default as Checkbox } from './checkbox';
export { default as Dialog } from './dialog';
export { default as FadeButton } from './fade-button';
export { default as Icon } from './icon';
export { default as Input } from './input';
export { default as InputGroup } from './input-group';
export { default as List } from './list';
export { default as Popover } from './popover';
// export { default as Portal } from './portal';
export { default as RadioButton } from './radio-button';
export { default as Select } from './select';
export { default as Search } from './search';
export { default as Switch } from './switch';
export { default as Tab } from './tab';
export { default as Tabset } from './tabset';
export { default as Textarea } from './textarea';
export { default as Tooltip } from './tooltip';
