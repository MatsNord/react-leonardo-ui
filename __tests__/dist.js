import React from 'react';
import renderer from 'react-test-renderer';

import { Button, Input } from '../dist/react-leonardo-ui';

test('Button from dist file renders correctly', () => {
  const tree = renderer.create(
    <Button>Text</Button>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Input from dist file renders correctly', () => {
  const tree = renderer.create(
    <Input value="Value" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
