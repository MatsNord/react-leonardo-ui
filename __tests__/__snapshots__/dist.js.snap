exports[`test Button from dist file renders correctly 1`] = `
<button
  className="lui-button">
  Text
</button>
`;

exports[`test Input from dist file renders correctly 1`] = `
<input
  className="lui-input"
  value="Value" />
`;
